Resources/Prototypes/_Goobstation/Entities/Objects/Weapons/Guns/Bow/hardlight_arrows.yml

```
@@ -177,10 +177,15 @@
     color: yellow
   - type: Projectile
-    deleteOnCollide: true
     damage:
       types:
         Heat: 0
   - type: SpawnEntitiesOnHit
+    spawnOnlyIfHitMob: true
     proto: MobAngryBeeWeak
     amount: 8
+    collideIgnoreEntities:
+    - MobBee
```

yamllint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)